protocol/messenger_local_backup.go

```diff
@@ -59,12 +58,22 @@ func (m *Messenger) ImportBackup(data []byte) error {
 		&backup,
 	)
 	if len(errs) > 0 {
-		return errors.Join(errs...)
+		// Log the errors but don't return as we want to process the rest of the backup anyway
+		// This allows partial restoration of the backup
+		// usually errors just mean that some data was already restored and is not needed to be restored again
+		for _, err := range errs {
+			m.logger.Error("Error restoring local backup from messenger", zap.Error(err))
```

Comment:
I think all the services should return the error as is. Then it should be the backup `core.importBackup` that logs all errors in a centra place, yet continues to iterate over other providers here:
https://github.com/status-im/status-go/blob/788e54c89c9a81ef39501511bad1afd855c07bb3/node/backup/core.go#L86-L88

I see that you want to distinguish an error to continue or not. Therefore you put the logging inside the service. But I think in case of importing there should be no errors that stops importing this backup for other providers. In other words, we should _always_ just log and continue.

So in this case you can just add the errors to the list, but only join&return this list in the very end of the function.